tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -166,12 +166,6 @@ void TangoRosNode::onInit() {
   area_description_T_start_of_service_publisher_ =
       node_handle_.advertise<geometry_msgs::TransformStamped>(
           AREA_DESCRIPTION_T_START_OF_SERVICE_TOPIC_NAME, queue_size, latching);
-  point_cloud_publisher_ =
```

Comment:
Correct. If a stream-like topic is advertised I would expect to see data on it. At least it simplifies reasoning on what a node is doing from the output of `rostopic list` or `rosnode info`.

An alternative would be to activate the image streams on demand once a subscriber connects, if that is possible without interruptions for other streams.